### ISSUE TITLE
Notebook content autosave

### ISSUE THREAD

**Issue by EvgeneKuklin** [oobabooga/text-generation-webui#5482]
I would like to see an option, so my notebook text from the last session would be autoloaded at the start of a new one. 
Or at least it would be stored somewhere so I would not lose it.
Also, it would be nice if generation data (model, params, etc.) would be stored too

**Comment by Tedy50** [comment]
Yes that woud be pretty useful. it is not fun to lose your work when you accidentally close the browser window.
Maybe there should be some rolling buffer of 100 files where older ones are automatically deleted because otherwise you will end with thousands of text files
